<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DailyLog</class>
 <widget class="QDialog" name="DailyLog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>768</width>
    <height>1366</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>768</width>
    <height>1366</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>768</width>
    <height>1366</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>幼圆</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>日志</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="comboBox">
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;幼圆&quot;;</string>
       </property>
       <item>
        <property name="text">
         <string>系统日志</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>报警日志</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>操作日志</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pB_Refresh">
       <property name="text">
        <string>刷新</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>ico/refresh.ico</normaloff>ico/refresh.ico</iconset>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pB_closeLog">
       <property name="text">
        <string>关闭</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
